### ISSUE TITLE
Notstrom Bug, bei Akkustatus größer 9

### ISSUE THREAD

**Issue by wiggal** [wiggal/GEN24_Ladesteuerung#120]
Hi, also irgendwie funktioniert das mit der Notstromgeschichte nicht richtig.

Aktuell habe ich {"5": "30", "6": "28", "7": "26", "8": "24", "9": "22", "10": "20"} drin stehen, aber er stellt mir immer 20% ein, obwohl die Prognose 6,6kWh anzeigt. (also nach der Scikitt-Learn-Optimierung)

Der Schalter steht auf 1, also so, dass er immer verstellen darf.

Habe ich das so richtig eingestellt, oder mache ich was falsch?  Die Reihenfolge der Werte ist egal, oder?

_Originally posted by @iReckyy in https://github.com/wiggal/GEN24_Ladesteuerung/discussions/101#discussioncomment-11588890_

**Comment by wiggal** [comment]
FIX ist im main [FIX: Sorierung Notstrom_Werte > 10](https://github.com/wiggal/GEN24_Ladesteuerung/commit/ebdc823207348fcb74f6fea3e81368028373a6bc)

**Comment by iReckyy** [comment]
Fix geladen, hat heute schonmal funktioniert.

**Comment by wiggal** [comment]
@iReckyy 
ist der Bug erledigt?

**Comment by iReckyy** [comment]
> @iReckyy ist der Bug erledigt?

Ja, der Notstrom-bug ist behoben, den Akkuschonung-Bug teste ich, wenn alles gut läuft, heute.

**Comment by wiggal** [comment]
Danke fürs Testen
